js/openfisca.js

```diff
@@ -29,6 +29,14 @@ function serialize(form) {
 		if (element.name == 'effectif_entreprise')
 			value ++
 
+		/* In the case of a `temps partiel`, we are asking hours per week,
+		the most common way to reason about it. But OpenFisca needs hours per month */
+		if (element.name == 'heures_remunerees_volume') {
+			var duree_legale_mensuelle = 151.66,
+				duree_legale_hebdomadaire = 35
```

Comment:
I'm not convinced at all about the interest of moving them far from their unique place of use.
Of course, the first problem is that they shouldn't be in the front but in the engine. But the engine needs serious work on the domain of *Temps de travail*. See the WIP PR openfisca/openfisca-france/pull/444